app/lib/dns.server.ts

```diff
@@ -165,6 +166,55 @@ async function checkDnsRecordExists(type: DnsRecordType, fqdn: string, value: st
   }
 }
 
+/**
+ * Get a full page of records from AWS Route53. If fqdn and recordType is specified,
+ * load the next page starting with that record
+ *
+ * https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-route-53/classes/listresourcerecordsetscommand.html
+ */
+export const getRecordPage = async (
```

Comment:
Should this be `getDnsRecordPage`?

Reply:
actually `getDnsRecordSetPage` now that you mention it :)